src/bots/depositor.py

```diff
@@ -75,9 +75,13 @@ def __init__(self, w3: Web3):
     def execute(self, block: BlockData) -> bool:
         self._check_balance()
 
-        module_id = get_preferred_to_deposit_module(self.w3, variables.DEPOSIT_MODULES_WHITELIST)
+        modules_id = get_preferred_to_deposit_modules(self.w3, variables.DEPOSIT_MODULES_WHITELIST)
 
-        if module_id:
+        if not modules_id:
+            # Read messages in case if no depositable modules for metrics
+            self.message_storage._receive_messages()
+
+        for module_id in modules_id:
```

Comment:
I'm not quite sure why we changed the logic from selecting one module to traversing the list, can you explain?

Reply:
If there no deposit messages for first module we will never deposit to others.
Now, we try deposit to each module that accepts deposits and is in whitelist

Reply:
Maybe add some logging when `modules_id` is empty?

Reply:
Added